install.sh

```diff
@@ -223,7 +251,10 @@ install_clisp() {
         sudo ln -s /usr/bin/clisp /usr/local/bin/clisp32
     else
         echo "Installing CLISP..."
-        sudo apt-get install clisp
+        case "$TRAVIS_OS_NAME" in
+            osx) brew install clisp ;;
```

Comment:
Building CLISP takes a long time. It'd be nicer to install a precompiled version. Do you know how to create casks? I guess a simple tarball would be fine too.